src/client/reducers/filters.js

```diff
@@ -0,0 +1,27 @@
+export default function (state = null, action) {
+    switch (action.type) {
+        case 'SET_ACTIVE_FILTER':
+            return state.map((filter) => {
+                if (action.payload.type !== filter.type) {
+                    filter.active = false;
+                } else {
+                    filter.active = true;
+                    filter.sortDir = action.payload.sortDir;
+                }
+                return filter;
+            });
+    }
+    
+    return [{
+        active: true,
+        title: 'by year',
+        type: 'release_year',
+        sortDir: true,
+    }, {
+        active: false,
+        title: 'by rating',
+        type: 'rating',
+        sortDir: true,
```

Comment:
@vitkuz this is excessive. there should be a filter  default store which contains filter description + active flag, which is a reference to the active filter. So on `SET_ACTIVE_FILTER` only active flag reference is changed